ui/src/components/view/Metrics.vue

```diff
@@ -0,0 +1,196 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+<template>
+  <div>
+    <a-row :gutter="12">
+      <a-col :md="24">
+        <a-card class="breadcrumb-card">
+          <a-row>
+            <a-col :span="24" style="padding-left: 12px">
+              <breadcrumb>
+                <a-tooltip placement="bottom" slot="end">
+                  <template slot="title">{{ $t('label.refresh') }}</template>
+                  <a-button
+                    style="margin-top: 4px"
+                    :loading="loading"
+                    shape="round"
+                    size="small"
+                    icon="reload"
+                    @click="fetchDetails()"
+                  >{{ $t('label.refresh') }}</a-button>
+                </a-tooltip>
+              </breadcrumb>
+            </a-col>
+          </a-row>
+        </a-card>
+      </a-col>
+    </a-row>
+    <a-divider/>
+    <a-row>
+      <a-col :md="24" :lg="12" :gutter="12">
+        <a-card>
+          <a-table
+            class="metric-card"
+            :columns="columns"
+            :loading="loading"
+            :data-source="dbMetrics"
+            :pagination="{ pageSize: 15}"
```

Comment:
Use pagination similar to other pages in UI (with option to select page size and left aligned) for consistency

Reply:
set to false, no pagination is desirable